native/cocos/platform/ios/modules/Battery.mm

```diff
@@ -29,6 +29,7 @@ of this software and associated engine source code (the "Software"), a limited,
 namespace cc {
 
 float Battery::getBatteryLevel() const {
+    [UIDevice currentDevice].batteryMonitoringEnabled = YES;
```

Comment:
Add some comments for better understanding

Reply:
done